### ISSUE TITLE
Update github actions CI workflow to also check code in doc comments

### ISSUE THREAD

**Issue by notmandatory** [bitcoindevkit/bdk#239]
Per comment in #236 the CI workflow needs to be updated to also check code in doc comments. Need to verify if it can be all done in one command or might need one fmt for regular code and one for code in docs.

```
cargo +nightly fmt -- --config format_code_in_doc_comments=true --check
```

**Comment by oleonardolima** [comment]
Hello @notmandatory 
I got interested in working on this issue, but it seems to have been already solved on #360, is that right?

**Comment by notmandatory** [comment]
Ah yes, good research! I will close this issue as it was fixed in #360.
